LiskHQ/lisk-core
# Update devnet configs - Closes #281

### What was the problem?

This PR resolves #281 

### How was it solved?

- Update devnet config to match latest development

### How was it tested?

- Remove data folder
- ./bin/run start -n devnet
- Check if there is no error, and start the application
